test/parallel/test-zlib-truncated.js

```diff
@@ -0,0 +1,50 @@
+'use strict';
+// tests zlib streams with truncated compressed input
+
+var common = require('../common');
+var assert = require('assert');
+var zlib = require ('zlib');
```

Comment:
Thanks for the input :)
Should I use 'let' for the scope-specific variables or is 'var' ok?